### ISSUE TITLE
about Performance

### ISSUE THREAD

**Issue by 6002tkfkd** [Rorozhl/MMKD#4]
Hello!

I ran the experiment shown in Table 1 (2nd row) where the teacher is ResNet32x4 and the student is MobileNetV2. The teacher achieved the expected accuracy (80.579%), but after applying KD, the student accuracy dropped significantly to 55.55%, which seems unusually low.

Here is the command I used to run the training:

python train_meta.py \
  --dataset cifar100 \
  --model_s MobileNetV2 \
  --teacher_num 3 \
  --ensemble_method META \
  --distill inter \
  --kd_T 4 \
  -r 1 -a 1 -b 100 \
  --hard_buffer \
  --epochs 240 \
  --learning_rate 0.1 \
  --lr_decay_epochs 150,180,210 \
  --lr_decay_rate 0.1 \
  --momentum 0.9 \
  --weight_decay 5e-4 \
  --meta_lr 1e-3 \
  --meta_freq 5 \
  --batch_size 64 \
  --gpu_id 0 \
  --trial 0

Is there anything I may have missed or should modify?
Any advice on why the student performance is much lower than expected would be greatly appreciated.

Thank you!
